libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss

```diff
@@ -0,0 +1,7 @@
+.no-margin {
+  margin: 0 !important;
```

Comment:
The `!important` doesn't seems necessary as it's only applyed to span tags

Reply:
In that case it is, because in another place the span margins are setted.

Reply:
who sets this margin?

Reply:
widgets-system/*/style.scss

pre span {
  &:not(:first-child) {
    margin-top: -18px;
  }
}